client/cz-registry.html

```diff
@@ -55,7 +55,10 @@
           }.bind(this));
         }
 
-        return this.registeredSources[type].then(source => source.registerQuery(query, callback));
+        return this.registeredSources[type].then(source => {
+          source.registerQuery(query, callback);
+          return source;
```

Comment:
Drawing attention to this change.
Is it weird to return a promise to the source element from registerSource()? I added this to support being able to call queryURL() on the issue source element to get an issue vendor specific URL.